packages/shared/components/molecules/AssetIcon.svelte

```diff
@@ -2,14 +2,14 @@
     import { NETWORK_ICON_SVG } from '@auxiliary/icon'
     import { getIconColorFromString } from '@core/account'
     import { COIN_TYPE, NetworkId } from '@core/network'
+    import { activeProfile } from '@core/profile'
     import { isBright } from '@core/utils'
-    import { ANIMATED_TOKEN_IDS, IPersistedAsset, NotVerifiedStatus, getAssetInitials } from '@core/wallet'
-    import { Animation, Icon, VerificationBadge } from 'shared/components'
+    import { ANIMATED_TOKEN_IDS, getAssetInitials, IPersistedAsset, TokenStandard } from '@core/wallet'
+    import { Animation, Icon, NetworkIconBadge } from 'shared/components'
```

Comment:
We should remove the VerificationBadge component, since we will handle verification in a different way.